lib/firmata.js

```diff
@@ -505,6 +523,12 @@ function Board(port, options, callback) {
     RES_TX3: 0x07,
   };
 
+  this.RESOLUTION = {
+    ADC: null,
```

Comment:
This is the only place in firmata.js where there is a reference to the resolution: https://github.com/firmata/firmata.js/blob/master/lib/firmata.js#L740. If we set defaults in firmata.js that param description could change to something like:

"The data to write to the pin between 0 and this.RESOLUTION.PWM."